src/main/java/thewaypointers/trafficsimulator/TrafficSimulatorStarter.java

```diff
@@ -9,7 +9,7 @@
 public class TrafficSimulatorStarter {
     final static long TIME_STEP = 100;
     final static long SIMULATION_TIME_STEP = 300;
-    final static String PATH_TO_MAP = "roadmaps/complex.xml";
+    final static String PATH_TO_MAP = "roadmaps/simple.xml";
```

Comment:
yea

On Mon, Mar 28, 2016 at 11:15 PM, Michał Szewczak <notifications@github.com>
wrote:

> In
> src/main/java/thewaypointers/trafficsimulator/TrafficSimulatorStarter.java
> <https://github.com/TheWaypointers/TrafficSimulator/pull/88#discussion_r57642892>
> :
>
> > @@ -9,7 +9,7 @@
> >  public class TrafficSimulatorStarter {
> >      final static long TIME_STEP = 100;
> >      final static long SIMULATION_TIME_STEP = 300;
> > -    final static String PATH_TO_MAP = "roadmaps/complex.xml";
> > +    final static String PATH_TO_MAP = "roadmaps/simple.xml";
>
> Could you change it back to Complex?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/TheWaypointers/TrafficSimulator/pull/88/files/03c95cb7d49f19bfa3f92098966cfd2f23e07e24#r57642892>
>